crates/sui-graphql-rpc/src/types/cursor.rs

```diff
@@ -105,6 +105,10 @@ pub(crate) trait Target<C: CursorType> {
     }
 }
 
+pub(crate) trait Checkpointed {
```

Comment:
Docs please!

Reply:
Covered later in the stack (sorry for deferring all these things)

Reply:
It's fine! But note that my earlier advise on landing a stack of PRs only works if the PRs are self-contained. If they are not, your best bet is to squash them into one PR and merge that.